src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -135,7 +138,7 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
     this.state.changeFields?.(fields.filter((f) => f.value !== ALL_VARIABLE_VALUE).map((f) => f.value!));
   }
 
-  private async updateBody(variable: CustomVariable) {
+  private updateBody(variable: CustomVariable = this.getVariable()) {
```

Comment:
Why are we adding default value here if all `updateBody` are passing variable?

Reply:
There was never the need to have that parameter. I actually removed it altogether.

Reply:
👍